src/test/java/ioanandroflorea/tests/Tema8/teste/DisabledDynamicButtonsTests.java

```diff
@@ -0,0 +1,27 @@
+package ioanandroflorea.tests.Tema8.teste;
+
+import ioanandroflorea.tests.Tema8.pageObjects.DisabledDynamicButtonsPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class DisabledDynamicButtonsTests extends BaseTest {
+
+    @Test(groups = {"mobile", "desktop"})
+    public void dynamicButtonsTests() {
+        driver.get("https://testpages.herokuapp.com/styled/dynamic-buttons-disabled.html");
+        DisabledDynamicButtonsPage disabledDynamicButtonsPage = new DisabledDynamicButtonsPage(driver);
+        disabledDynamicButtonsPage.waitForTheStartButtonToBeClickable();
+        disabledDynamicButtonsPage.clickOnStartButton();
+        disabledDynamicButtonsPage.waitForOneButtonToBeClickable();
+        disabledDynamicButtonsPage.clickOnOneButton();
+        disabledDynamicButtonsPage.waitForTheMessageToDisappear();
+        disabledDynamicButtonsPage.clickOnSecondButton();
+        disabledDynamicButtonsPage.waitForTheMessageToDisappear();
+        disabledDynamicButtonsPage.clickOnThirdButton();
+        Assert.assertEquals(disabledDynamicButtonsPage.confirmationMessage(), "Click Buttons In Order", "All Buttons Clicked");
```

Comment:
The expected text should be "All Buttons Clicked" and not "Click Buttons In Order". Please make this change. Thanks!

Reply:
I changed it, but the test still fails. I don't know what I should do.